raster/r.mapcalc/map3.c

```diff
@@ -18,6 +18,7 @@
 /****************************************************************************/
 
 static void prepare_region_from_maps(expression **, int, int);
+int columns;
```

Comment:
This does not look right given that rows and depths are used in the same way. Maybe it is not needed after all?

Reply:
You're right. Interesting. Let me test it.

Reply:
`rows` is defined in `evaluate.c`, but `columns` is defined individually in `map.c`. He used this same pattern for `map3.c`. Without this change, `r3.mapcalc` fails to build.

Reply:
Moving it to `evaluate.c`